web/src/lib/components/NostrChat.svelte

```diff
@@ -154,9 +178,84 @@
         })
     }
 
+    function queryNoteInformationInBatches() {
+        let noteInfoToGetLocal: string[] = [];
+
+        let i=0;
+
+        for (const [key, note] of notesMap) {
+            if (note === null) {
+                notesMap.set(key, true);
+                noteInfoToGetLocal.push(key);
+                i++;
+
+                if (i == nostrQueriesBatchSize) {
+                    break;
+                }
+            }
+        }
+
+        if (noteInfoToGetLocal.length === 0) {
+            return;
+        }
+
+        pool.relays.forEach(relay => {
+            let filter: Filter = {
+                kinds: [
+                    nostrEventKinds.note,
+                    nostrEventKinds.replies,
+                    nostrEventKinds.reactions,
+                ],
+                '#e': noteInfoToGetLocal
+            };
+
+            const sub: Sub = relay.sub([filter]);
+
+            sub.on('event', event => {
+                const kind = event.kind;
+
+                // TODO: REPLIES AND LIKES
+
+                if (kind === 7) {
```

Comment:
Add a constant for the magic number!

Reply:
That's fixed in the next PR already, using the new Enum for types.